.github/workflows/pipeline-build-linux.yml

```diff
@@ -66,27 +63,46 @@ jobs:
             echo "RI_APP_TYPE=${{ env.RI_APP_TYPE }}"
           } >> "${{ env.envFile }}"
 
-      - name: Build linux package (production)
+      - name: Download backend x64
+        uses: ./.github/actions/download-backend
+
+      - name: Build linux package x64 (production)
+        if: inputs.environment == 'production'
+        run: |
+          yarn package:prod --target linux-x64 --out ${packagePath}-x64.vsix
+
+      - name: Build linux package x64 (staging)
+        if: inputs.environment == 'staging'
+        run: |
+          sed -i "s/^RI_APP_FOLDER_NAME=.*/RI_APP_FOLDER_NAME='.redis-for-vscode-stage'/" ${{ env.envFile }}
+          yarn package:stage --target linux-x64 --out ${packagePath}-x64.vsix
+
+      - name: Download backend arm64
```

Comment:
This is a general note, and I don't think it's slow right now as well, but we can parallelize the ARM and AMD builds, because they are independent (correct me if I'm wrong).

Reply:
some space for improvements for the future